gaphor/core/modeling/tests/test_elementfactory.py

```diff
@@ -40,7 +49,7 @@ def test_flush(factory):
 
     gc.collect()
 
-    assert len(list(factory.values())) == 0, list(factory.values())
+    assert not list(factory.values()), list(factory.values())
```

Comment:
I like this change to be more readable :+1:

Reply:
Thanks to Sourcery :wink: